projects/xerces-c/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER vincent.ulitzsch@live.de
+RUN apt-get update && apt-get install -y make autoconf automake libtool wget zlib1g-dev
+RUN apt-get update && \
+apt-get install -y  libtool ninja-build cmake subversion
+RUN apt-get install -y libstdc++6
+RUN svn co https://svn.apache.org/repos/asf/xerces/c/trunk $SRC/xerces-c
+RUN git clone --depth 1 https://github.com/google/libprotobuf-mutator.git
+RUN (mkdir LPM && cd LPM && cmake ../libprotobuf-mutator -GNinja -DLIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF=ON -DLIB_PROTO_MUTATOR_TESTING=OFF -DCMAKE_BUILD_TYPE=Release && ninja)
```

Comment:
So LPM isn't MSAN clean now, but building it here will ensure this project will not work with MSAN if it does become MSAN clean. Building in the `build.sh` would be better in that scenario since it would build with the MSAN flags in an MSAN build (as you may know all dependencies of a project need to use MSAN for MSAN to work).

I'm fine with this for now since LPM doesn't work with MSAN and so this is faster (since LPM only needs to be built once instead of in each fuzzer build), just thought I'd let you know.